src/components/Card.js

```diff
@@ -39,7 +39,12 @@ const FlashCard = ({ card, classes }) => {
 
 export default withStyles(styles)(FlashCard); 
 
-//TODO From here I need to finish that quick youtube video and add a flipping card 
+// ! Problem - when the cards are flipped (clicked) they change width, and cause their parent to collapse - it's an ugly flick in the UI 
```

Comment:
There are a few ways I think I can solve this, but I'm not sure which path to take 🤔